apidoc/Annotation.yml

```diff
@@ -76,78 +76,86 @@ properties:
     type: Titanium.UI.View
     since: "3.1.0"
 
-  - name: showMarker
+  - name: showAsMarker
     summary: |
         Boolean to show an annotation view that displays a balloon-shaped 
         marker at the designated location.
     description: |
         Must be set during creation. This is ignored if the customView property is set.
+        See the example "Map Example With Marker Annotation and Clustering".
     type: Boolean
     default: false
     platforms: [iphone, ipad]
     osver: {ios: {min: "11.0"} }
+    since: "6.3.0"
 
   - name: markerGlyphText
     summary: The text to display in the marker balloon.
     description: |
-        Use this property or the markerImage property to specify the marker balloon content. 
+        Use this property or the <Modules.Map.Annotation.markerGlyphImage> property to specify the marker balloon content. 
         If you specify both an image and text, the text is displayed.
         The amount of space available for displaying your glyph text is limited. Specify no more than 
         two or three characters for any strings you assign to this property.
     type: String
     platforms: [iphone, ipad]
     osver: {ios: {min: "11.0"} }
+    since: "6.3.0"
 
   - name: markerGlyphColor
     summary: The color to apply to the glyph text or image.
     description: |
-        The default value of this property is nil, which applies the standard tint color for the 
+        The default value of this property is undefined, which applies the standard tint color for the 
         current map style.
-    type: [Number, String]
```

Comment:
So not both is possible?

Reply:
No, we should use similar to backgroundColor etc, where we pass String only. In case of pincolor only Number is passed as in < iOS 9 it support the enum value.